inst/extdata/packages/ENMTools.yaml

```diff
@@ -0,0 +1,32 @@
+name: ENMTools
+title: "Analysis of Niche Evolution using Niche and Distribution Models"
+version: 1.1.1
+author: "Dan Warren and Russell Dunnage"
```

Comment:
```suggestion
author: "Dan Warren, Russell Dunnage"
```

Reply:
His last name is actually Dinnage, and I swear I corrected that before I submitted it.  Dammit.

Reply:
This is problematic as we use CRAN metadata to validate the package metadata.

Reply:
In this case if you want to use metadata different from CRAN you need to change the cran field to no to stop automatic checking.